quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -527,4 +699,17 @@ impl BbrCongestionController {
 
         self.cwnd = self.cwnd.max(self.prior_cwnd);
     }
+
+    /// Modulates the congestion window based on newly lost bytes
+    #[inline]
+    fn modulate_cwnd_for_recovery(&mut self, lost_bytes: u32) {
+        //= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.6.4.4
+        //# BBRModulateCwndForRecovery():
+        //#   if (rs.newly_lost > 0)
+        //#     cwnd = max(cwnd - rs.newly_lost, 1)
```

Comment:
This isn't the `minimum_window`, it's `1`. Is there a reason for the change?

Reply:
The 1 would be 1 packet in this case, but that doesn't make sense to me regardless, since the congestion window should not drop below the minimum. I couldn't find the implementation of this in the Linux BBRv2 implementation or Chromium, so I'm not sure exactly what the thinking was here. I think sticking with a lower bound of the minimum window (which is 4 packets) should be safe.

Reply:
Ok. Do you think it's worth opening an issue with the RFC?

Reply:
Actually I found it in the BBR v1 Linux implementation: https://github.com/torvalds/linux/blob/master/net/ipv4/tcp_bbr.c#L491-L492

I'm still not sure why its using 1 here. I can add it to [my list of issues](https://groups.google.com/g/bbr-dev/c/xmley7VkeoE), but I haven't gotten a response yet from the BBR authors